app/client/doc/commands/client.md

```diff
@@ -9,8 +9,9 @@ The CLI is meant to be an user but also a machine friendly way for interacting w
 ### Options
 
 ```
-      --data_dir string         Path to store pocket related data (keybase etc.) (default "/Users/olshansky/.pocket")
+      --data_dir string         Path to store pocket related data (keybase etc.) (default "/home/alex/.pocket")
```

Comment:
lol

Reply:
It takes the current user directory. 🤷‍♂️

Anyone updating it would end up with the same result.

Lemme know how do you want to handle this. If we want it to be stable we'd have to be creative, maybe some `sed`  wizardry in the `Makefile` 🤔 

it could be a bounty I guess.